package.json

```diff
@@ -30,10 +29,11 @@
     "ci:publint": "turbo run ci:publint",
     "ci:sherif": "sherif --ignore-package \"./examples/*\" --ignore-package \"./websites/*\" --ignore-package \"./docs/*\" --ignore-dependency \"@tanstack/react-query\" ",
     "ci:test": "turbo run ci:test",
-    "ci:test:watch": "turbo run ci:test:watch --parallel",
     "ci:type": "turbo run ci:type",
     "clean": "turbo run clean",
-    "dev": "turbo run dev --concurrency 20",
+    "dev": "concurrently \"pnpm dev:build\" \"pnpm dev:test\"",
+    "dev:build": "turbo watch build --filter=./packages/**",
```

Comment:
```suggestion
    "dev": "concurrently \"pnpm dev:prepack\" \"pnpm dev:test\"",
    "dev:prepack": "turbo watch prepack",
```

Reply:
That's such a great idea I didn't expect a double build, but thanks!